apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```
@@ -444,14 +299,30 @@ export const IndictmentCount: FC<Props> = ({
+      ? updatedOffenses
+      : update.deprecatedOffenses
+    const hasUpdatedDeprecatedSubstances = isOffenseEndpointEnabled
+      ? false
+      : update.substances
+    if (hasUpdatedOffenses || hasUpdatedDeprecatedSubstances) {
+      const offenses = isOffenseEndpointEnabled
+        ? updatedOffenses?.map((o) => o.offense)
+        : update.deprecatedOffenses || indictmentCount.deprecatedOffenses
+      const substances = isOffenseEndpointEnabled
+        ? updatedOffenses?.reduce(
+            (res, offense) => ({ ...res, ...offense.substances }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)